src/angular-core/styles/typography/_link.scss

```diff
@@ -3,22 +3,10 @@
 
 @mixin standardLink {
   color: $sbbColorBlack;
-  line-height: pxToEm(23px);
+  line-height: pxToRem(23px);
   text-decoration: underline;
   text-decoration-skip-ink: auto; // -> read more about here https://medium.com/@iamhiwelo/improving-text-readability-for-dyslexic-users-with-skip-ink-underlines-bf52a2f3426b
 
-  @include publicOnly() {
-    @include mq($from: desktop4k) {
-      font-size: pxToRem($sizeFontDefault4k);
```

Comment:
I think, we still need the font-size part here? As it is now, there is no 4k and 5k font-size